templates/ords-remix-jwt-sample/app/utils/auth.server.ts

```diff
@@ -45,7 +45,7 @@ const auth0Strategy = new Auth0Strategy(
     clientSecret: AUTH0_CLIENT_SECRET,
     domain: AUTH0_DOMAIN,
     audience: AUTH0_AUDIENCE,
-    scope: ['openid', 'email', 'profile', 'concert_app_euser', 'concert_app_authuser', 'concert_app_admin', 'oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_VIEW', 'oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_ARTIST_VIEW', 'oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_VENUES_VIEW'],
+    scope: ['openid', 'email', 'profile', 'concert_app_authuser', 'concert_app_admin', 'oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_VIEW', 'oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_ARTIST_VIEW', 'oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_VENUES_VIEW'],
```

Comment:
please also remove the `oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_VIEW`, `oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_ARTIST_VIEW`, `oracle.dbtools.autorest.privilege.CONCERT_SAMPLE_APP.SEARCH_VENUES_VIEW` scopes since those privileges are no longer needed.

Reply:
Privileges removed.